packages/pangea-sdk/pangea/services/vault/models/common.py

```diff
@@ -148,40 +170,47 @@ class GetRequest(APIRequestModel):
     verbose: Optional[bool] = None
 
 
-class CommonGetResult(PangeaResponseResult):
+class ItemVersionData(PangeaResponseResult):
+    version: int
+    state: str
+    created_at: str
+    public_key: Optional[EncodedPublicKey] = None
+    secret: Optional[str] = None
+
+
+class GetResult(PangeaResponseResult):
     type: str
     id: str
-    version: int
     name: Optional[str] = None
     folder: Optional[str] = None
     metadata: Optional[Metadata] = None
     tags: Optional[Tags] = None
-    auto_rotate: Optional[bool] = None
-    rotation_policy: Optional[str] = None
-    last_rotated: Optional[str] = None  # TODO: should be time
-    next_rotation: Optional[str] = None  # TODO: should be time
-    expiration: Optional[str] = None  # TODO: should be time
-    created_at: Optional[str] = None  # TODO: should be time
-    revoked_at: Optional[str] = None  # TODO: should be time
-    retain_previous_version: Optional[bool] = None
+    rotation_frequency: Optional[str] = None
+    rotation_state: Optional[str] = None
+    last_rotated: Optional[str] = None
+    next_rotation: Optional[str] = None
+    expiration: Optional[str] = None
+    destroyed_at: Optional[str] = None
+    algorithm: Optional[Union[AsymmetricAlgorithm, SymmetricAlgorithm]] = None
+    purpose: Optional[KeyPurpose] = None
+    versions: List[ItemVersionData] = []
 
 
 class ListItemData(APIRequestModel):
     id: str
     type: str
-    managed: bool
-    last_rotated: Optional[str] = None  # TODO: should be time
-    next_rotation: Optional[str] = None  # TODO: should be time
-    expiration: Optional[str] = None  # TODO: should be time
-    rotation_policy: Optional[str] = None
+    last_rotated: Optional[str] = None
+    next_rotation: Optional[str] = None
+    expiration: Optional[str] = None
+    rotation_frequency: Optional[str] = None
     identity: str
     version: int
```

Comment:
Please change "version" to "versions" object according to new specs.

Reply:
fixed